docs/_packages/oauth.md

```diff
@@ -165,15 +171,39 @@ const installer = new InstallProvider({
     // returns nothing
     storeInstallation: (installation) => {
```

Comment:
nit: `async` has been missing. It's the same for `storeOrgInstallation`
```suggestion
    storeInstallation: async (installation) => {
```
https://github.com/slackapi/node-slack-sdk/blob/2d7a94c4f342fac94c7f6eb6be01b1257ce1df4e/packages/oauth/src/index.ts#L443